packages/grafana-ui/src/components/TimePicker/TimePickerContent/TimePickerCalendar.tsx

```diff
@@ -129,6 +128,7 @@ const getBodyStyles = stylesFactory((theme: GrafanaTheme) => {
       .react-calendar__tile--now {
         margin-bottom: 4px;
         background-color: inherit;
+        height: 26px;
```

Comment:
Just out of curiosity, are these changes necessary for this fix?

Reply:
Ah - forgot to add that in the description. This fixes some weirdness in when selecting several dates.